### ISSUE TITLE
Trigger Collective onboarding emails for new admins

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#5230]
Currently when a new Collective starts up, we send the admins a series of onboarding emails like the below. But if a new admin is added later, they miss out on all this info. I think if a new admin joins a Collective, they should also receive these emails, counting days since they accepted the admin invite instead of days since the Collective was created.

![Screen Shot 2022-02-14 at 8 56 12 PM](https://user-images.githubusercontent.com/5498878/153823080-fc2f5e64-1ad4-44d1-915b-4b1ebcab242d.png)

![Screen Shot 2022-02-14 at 8 54 57 PM](https://user-images.githubusercontent.com/5498878/153823093-fe639e31-1a00-4649-921b-daf9d704eca8.png)

**Comment by BenJam** [comment]
My immediate questions here are:

- how many collectives invite an admin say, a week after they have started?
- how relevant are the notifications to a new admin (as opposed to the admin that set up the collective)?

## Impact

There are over 2k project that have added an admin a week after creating their project (just under 2k a month after starting, and just under 800 a year after):

```
SELECT COUNT(*), c."name" n
FROM "Members" m
JOIN "Collectives" c on m."CollectiveId" = c."id"
WHERE m."createdAt" > c."createdAt" + '1 week'
AND m."role" = 'ADMIN'
GROUP BY n
```

## Onboarding email content

### [Day 2](https://github.com/opencollective/opencollective-api/blob/main/templates/emails/onboarding.day2.foundation.hbs)
we ask the admin to 
- fill in their about section
- add tags
- share your page
- embed the contrib flow elsewhere
- invite them to learn about expenses

### [Day 3](https://github.com/opencollective/opencollective-api/blob/main/templates/emails/onboarding.day3.foundation.hbs)
On day three we 
- encourage people to send updates
- check out other groups using Open Collective 
- direct people to official docs (tax, accounting banking etc)
- remind people of the rules and policies
- reminder of some of the services that are offered (virtual cards, liability insurance etc)

### [Day 7](https://github.com/opencollective/opencollective-api/blob/main/templates/emails/onboarding.day7.hbs)
we ask admins
- have you customised your contribution tiers?
- remind folks about events, expenses, goals and connected collective features

### [If three are no expenses](https://github.com/opencollective/opencollective-api/blob/main/templates/emails/onboarding.noExpenses.hbs)
We send an email encouraging people to submit and invite expenses

### [If there are no updates](https://github.com/opencollective/opencollective-api/blob/main/templates/emails/onboarding.noUpdates.hbs)
We encourage them to publish an update and direct them to the documentation 

## Separating the content 

As @iamronen [points out](https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2992288) we may want to instead create a new notification chain for a new administrator, who may be joining an active project. 

Some of that content might be the same but, I can imagine that sending an email to a new admin of an established project to 'update your about page' will be seen as us being a bit brainless. Some things that we might want to share with a new admin:

### Day 2
- expenses processes, how to approve, request more information etc.
- tagging, projects and other budget management tools 

### Day 3
- similar to the day three above we may want to remind folks about the policies that they're working under

### Day 7
- We may want to remind them about the other services that are on offer, depending on whether that collective is already using them

## Next Steps

I think we've established that this is an issue broadly, and proposed that perhaps we need another onboarding drip campaign for admins invited rather than new collectives created. 

@Memo-Es should we consider that a part of the #4829 project, or does that not touch the drip campaign emails that are sent?

**Comment by Memo-Es** [comment]
I agree with @BenJam. We might want to create a new series of onboarding emails specifically for new admins of an existing Collective; the work then is to craft the content of these emails and add the respective templates in the database.

**Comment by kaylarep** [comment]
+1 to this issue.  It's important that new Admins learn about platform usage, and since we don't manually monitor when new Admins are added, automated-emails would be very helpful to onboard new Admins. (Though we at least find when new Admins are added within the Activity log now)

If a fiscal host could customize these automated messages too, that would be great.

### Context:
An OCF-hosted group had the Admin responsible for that Collective's virtual card leave the Collective. The new Admin responsible for the card did not know what to do at all.

Having an email sent to them once they were added as an Admin to this collective would improve their platform experience. 

A customized message would be nice so that we could also include something in the automated message that 1) invites them to schedule an onboarding call with our Program Director 2) invite them to our Monthly Welcome meeting

### To consider:
- would this only be necessary/important for fiscally-sponsored Collectives?
    - New accounts are already sent some automated message upon creating an account.   BUT its important for Admins of fiscally-sponsored projects to understand the rules/limitations of the their fiscal sponsor, and right now, newly added Admins might be under the radar and missing out on important compliance information.
